src/app/components/character-count/index.html

```diff
@@ -0,0 +1,284 @@
+              >
+                Do use character count on input fields with character limits that a user could realistically exceed.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+
+        <h3
+          class="sky-text-danger"
+        >
+          Don't use when
+        </h3>
+
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Don’t use character count on inputs where the character limit is apparent or implied by the data type.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-2.png"
+              />
+              <div
+                class="caption caption-danger sky-text-danger"
+              >
+                Don't use character count when the content has an expected or standard length.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              Don’t use character count on inputs when exceeding the character limit is unrealistic. Use normal form validation to handle errors caused by character limit violations.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div
+              class="sky-margin-stacked-content-section"
+            >
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-3.png"
+              />
+              <div
+                class="caption caption-danger sky-text-danger"
+              >
+                Don't use character count when the character limit is unlikely to be exceeded.
+              </div>
+            </div>
+
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-usage-4.png"
+              />
+              <div
+                class="caption caption-success sky-text-success"
+              >
+                Do use normal form validation for edge cases where a high character limit is exceeded.
+              </div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+
+      <sky-demo-page-content sectionHeading="Anatomy">
+
+        <sky-docs-anatomy
+          imagePath="~/assets/img/guidelines/character-count/character-count-anatomy-1.png"
+        >
+          <sky-docs-anatomy-item
+            anatomyNumber="1"
+          >
+            Input field
+          </sky-docs-anatomy-item>
+          <sky-docs-anatomy-item
+            anatomyNumber="2"
+          >
+            Character count/limit
```

Comment:
What do you think of changing this to `Character count indicator`? If that doesn't work, I'd change `/` to `and`.

Reply:
Agreed.